docker-app/qfieldcloud/core/templates/admin/login.html

```diff
@@ -0,0 +1,8 @@
+{% extends "admin/login.html" %}
+
+{% block content %}
+    {{ block.super }}
+    {% if request.GET.sso %}
```

Comment:
```suggestion
    {% comment %}TODO add a configuration based mechanism to render the social login instead of depending on query param {% endcomment %}
    {% if request.GET.sso %}
```
I guess we want to add such TODO here?